server/lib/whispers/whisper-models.ts

```diff
@@ -0,0 +1,167 @@
+import sql from 'sql-template-strings'
+import { WhisperMessageData } from '../../../common/whispers'
+import db from '../db'
+
+export interface WhisperSessionEntry {
+  targetUserId: number
+  targetUserName: string
+}
+
+export async function getWhisperSessionsForUser(userId: number): Promise<WhisperSessionEntry[]> {
+  const { client, done } = await db()
+  try {
+    const result = await client.query(sql`
```

Comment:
Generally you always want to pass a generic parameter to `client.query`, otherwise it's `any` and that's bad for typing anything further than that in the function.

Apply throughout this file